velocore/outputData.csv

```diff
@@ -0,0 +1,19 @@
+block_number,timestamp,user_address,token_address,token_balance
+2999728,2024-03-19T14:01:25.000Z,0x00c8d3f9a1d4ae64d0eecd04e837d007fd36d04d,0x176211869ca2b568f2a7d4ee941e073a821ee1ff,11927
+2999728,2024-03-19T14:01:25.000Z,0x00c8d3f9a1d4ae64d0eecd04e837d007fd36d04d,0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee,12382797862820
+2999728,2024-03-19T14:01:25.000Z,0x00c8d3f9a1d4ae64d0eecd04e837d007fd36d04d,0xa219439258ca9da29e9cc4ce5596924745e12b93,20696
+2999728,2024-03-19T14:01:25.000Z,0x00c8d3f9a1d4ae64d0eecd04e837d007fd36d04d,0x7d43aabc515c356145049227cee54b608342c0ad,31003526088695939
+2999728,2024-03-19T14:01:25.000Z,0x00d2ba56ae934f85e028bfe191310dd67a3fe3b8,0x176211869ca2b568f2a7d4ee941e073a821ee1ff,28116207
+2999728,2024-03-19T14:01:25.000Z,0x014603be2f4e4baa0c69945559c3f6b422ab8796,0x176211869ca2b568f2a7d4ee941e073a821ee1ff,3677135
+2999728,2024-03-19T14:01:25.000Z,0x014603be2f4e4baa0c69945559c3f6b422ab8796,0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee,903664795290366
+2999728,2024-03-19T14:01:25.000Z,0x014603be2f4e4baa0c69945559c3f6b422ab8796,0x3f006b0493ff32b33be2809367f5f6722cb84a7b,302654
+2999728,2024-03-19T14:01:25.000Z,0x0283d6d9248dcbac2aaea05444795017e2ac5558,0x176211869ca2b568f2a7d4ee941e073a821ee1ff,1000551
+2999728,2024-03-19T14:01:25.000Z,0x0283d6d9248dcbac2aaea05444795017e2ac5558,0x7d43aabc515c356145049227cee54b608342c0ad,1000998194370630993
+2999728,2024-03-19T14:01:25.000Z,0x02aa04217f687611de085ee878d10b66abeecde6,0x176211869ca2b568f2a7d4ee941e073a821ee1ff,28482007
+2999728,2024-03-19T14:01:25.000Z,0x037dbeb376900653e8b1744ccfd30f074f9f282a,0x176211869ca2b568f2a7d4ee941e073a821ee1ff,2111344
+2999728,2024-03-19T14:01:25.000Z,0x037dbeb376900653e8b1744ccfd30f074f9f282a,0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee,50880950049572
+2999728,2024-03-19T14:01:25.000Z,0x039f6ec840680b527dc8939064ee25f4ade318d7,0x176211869ca2b568f2a7d4ee941e073a821ee1ff,26138487
+2999728,2024-03-19T14:01:25.000Z,0x039f6ec840680b527dc8939064ee25f4ade318d7,0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee,12057861298705141
+2999728,2024-03-19T14:01:25.000Z,0x039f6ec840680b527dc8939064ee25f4ade318d7,0xb5bedd42000b71fdde22d3ee8a79bd49a568fc8f,10480007178351956
+2999728,2024-03-19T14:01:25.000Z,0x039f6ec840680b527dc8939064ee25f4ade318d7,0x7d43aabc515c356145049227cee54b608342c0ad,26208754629527261094
+2999728,2024-03-19T14:01:25.000Z,0x03ddd5a675e2cfa35b9ae2b207a7dbd95fe24646,0x176211869ca2b568f2a7d4ee941e073a821ee1ff,1000002
```

Comment:
This is just a sample outputData.csv file.
Note that the token address 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee is the Ethereum address.